reportportal/README.md

```diff
@@ -80,3 +80,82 @@ mongodb:
     address: mongodb://mongodb.default.svc.cluster.local
     port: 27017
 ```
+
+### Installation notes
+1. Make sure you have Kubernetes up and running
+2. Reportportal requires installed [mongodb](https://github.com/helm/charts/tree/master/stable/mongodb) and [elasticsearch](https://github.com/helm/charts/tree/master/stable/elasticsearch) to run. Required versions of helm charts are described in requirements.yaml
+If you don't have your own mongodb and elasticsearch you can install them from helm charts. 
+
+For example to install mongodb please use this commands:
+```sh
+helm dependency build ./reportportal/
+helm install ./reportportal/charts/mongodb-0.4.18.tgz
+```
+After deploying mongodb please copy address and port from output notes. Should be something like this:
+```
+NOTES:
+MongoDB can be accessed via port 27017 on the following DNS name from within your cluster:
+ill-vulture-mongodb.default.svc.cluster.local
+```
+Elasticsearch chart should be installed in the same manner:
+```sh
+helm install ./reportportal/charts/elasticsearch-1.17.0.tgz
+```
+
+3. After mongodb and elasticsearch up and running you should edit values.yaml to adjust reportportal settings.
```

Comment:
3. After mongodb and elasticsearch are up and running, edit values.yaml to adjust ReportPortal settings.